src/replica/HttpConfigurationModule.cc

```diff
@@ -275,17 +275,22 @@ json HttpConfigurationModule::_deleteTable() {
 json HttpConfigurationModule::_addTable() {
     debug(__func__);
 
-    auto const database = body().required<string>("database");
-    auto const table = body().required<string>("name");
-    auto const isPartitioned = body().required<int>("is_partitioned") != 0;
+    // FIXME: extend the service to accept all attributes of the table.
+    // Create a method in the base class to extract standard attributes from
+    // from the requests's body. Use this method here and in the method HttpIngestModule::_addTable
```

Comment:
IS there a Jira ticket for this FIXME?

Reply:
I don't have (yet) any JIRA ticket for that. I will have the one later. I have registered this concern in the refactoring & code improvement list for the Replication/Ingest system at: https://confluence.lsstcorp.org/display/DM/Completed%2C+on-going+and+proposed+improvements